website/docs/guides/hyper_file/create_update.md

```diff
@@ -41,7 +41,7 @@ with HyperProcess(Telemetry.SEND_USAGE_DATA_TO_TABLEAU,
 The script consist of 3 high-level steps:
 
 1. Start a Hyper process. The [`HyperProcess`](../../hyper-api/hyper_process)
-2. Create a connection to the `.hyper` file. Since we create the [`Connection` class](../../hyper-api/connection) class with the `CreateMode.CREATE_AND_REPLACE`, the `.hyper` will be automatically created if doesn't exist yet, and will be overwritten if a file with that name already exists.
+2. Create a connection to the `.hyper` file. Since we create the [`Connection`](../../hyper-api/connection) class with the `CreateMode.CREATE_AND_REPLACE`, the `.hyper` will be automatically created if doesn't exist yet, and will be overwritten if a file with that name already exists.
```

Comment:
```suggestion
2. Create a connection to the `.hyper` file. Since we create the [`Connection`](../../hyper-api/connection) class with the `CreateMode.CREATE_AND_REPLACE`, the `.hyper` file will be automatically created if it does not exist yet, and will be overwritten if a file with that name already exists.
```